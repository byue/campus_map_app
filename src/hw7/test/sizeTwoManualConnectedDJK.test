# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 2 connected graph.

CreateGraph g

AddNode g a
AddNode g b

AddEdge g a b 3.512442

FindPath g a b