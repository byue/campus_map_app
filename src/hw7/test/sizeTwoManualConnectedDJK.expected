# Simple test of Djikstra's algorithm.
# Verifies output of DJK for size 2 connected graph.

created graph g

added node a to g
added node b to g

added edge 3.512 from a to b in g

path from a to b:
a to b with weight 3.512
total cost: 3.512